Add visits today sensor for pets to Litter-Robot documentation (#39698)
home-assistant/home-assistant.github.io: source/_integrations/litterrobot.markdown

### Pet

| Entity       | Domain   | Description                                       |
| ------------ | -------- | ------------------------------------------------- |

> | Visits today | `sensor` | Pet's daily visits to an associated Litter-Robot. |

Reviewer: ```suggestion
| Visits today | `sensor` | Pet's daily visits to the Litter-Robot. |
```
does "associated" have any special meaning? otherwise I think we can simplify
